cli/cmd/root.go

```diff
@@ -47,7 +47,7 @@ func NewRootCmd() *cobra.Command {
 	must(rootCmd.MarkPersistentFlagFilename("config", "yaml"))
 
 	rootCmd.PersistentFlags().Bool("debug", false, "enable debug logging")
-	rootCmd.PersistentFlags().Bool("force", false, "disables version validation errors - might result in corrupted clusters")
+	rootCmd.PersistentFlags().Bool("force", false, "disable version compatibility checks - might result in corrupted clusters")
```

Comment:
The global --force flag might get a broader meaning in the future. Would this be okay? Otherwise, we should rename it now.